test/unit/io/detail/detail_record_test.cpp

```diff
@@ -12,7 +12,7 @@
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
 #include <seqan3/alphabet/quality/phred42.hpp>
 #include <seqan3/io/detail/record.hpp>
-#include <seqan3/range/views/zip.hpp>
+#include <seqan3/utility/views/zip.hpp>
 
 #include <seqan3/test/expect_same_type.hpp>
 
```

Comment:
I think it is nicer, if the seqan/test includes are seperated, as here. But I found no rule for this. @eseiler?
https://github.com/seqan/seqan3/wiki/File-structure-and-naming

Reply:
Every include that starts with `#include <seqan3/` should go into the same block, so `utility` comes after `test`